server/server.js

```diff
@@ -4,8 +4,11 @@ import app from "./app";
 import { connectDb, disconnectDb } from "./db";
 import config from "./utils/config";
 import logger from "./utils/logger";
-
+import { Logger, LogOutput } from "concurrently";
+import { init } from "./socket";
 const server = http.createServer(app);
+init(server);
```

Comment:
let's rename this init to initSocket to make it more readable (needs to happen in socket.js and in the import too, obviously)

Reply:
done